.circleci/config.yml

```diff
@@ -31,24 +84,39 @@ jobs:
 
     steps:
       - checkout
-      - run: bundle install
-      - run: yarn install
-      - run: git config --global user.email "ci-build@coopdevs.org"
-      - run: git config --global user.name "CI"
-      - add_ssh_keys:
-          fingerprints:
-            - "f6:8f:6b:93:bf:a3:a1:52:46:c6:2b:54:b6:5c:2e:23"
-      - run: git config -l
-      - run: ./deploy
+      - *restore_bundler_cache
+      - *bundle_install
+      - *save_bundler_cache
+      - *restore_yarn_cache
+      - *yarn_install
+      - *save_yarn_cache
+      - *git_config
+      - *add_ssh_keys
+      - *git_config_l
+      - *deploy
+
 
 workflows:
   version: 2
-  build-and-deploy:
+
+  feature:
+    jobs:
+      - build:
+          <<: *feature_branch_filter
+  develop:
+    jobs:
+      - build:
+          <<: *develop_branch_filter
+      - deploy:
+          <<: *develop_branch_filter
+          requires:
+            - build
+  # Deploy master branch to production environment
+  production:
```

Comment:
we don't need this block as deploys are performed from `develop` already. That's because deployment involves storing the Jekyll build artifacts in the `master` branch so Github Pages picks them up.

Reply:
I'm not sure I understand. Are you saying that deployments happen from `develop` and `master` as a single process?

Reply:
Ah, I just looked at the deploy script. It appears that you initiate the build process in `develop` and the deploy script pushes those changes to `master`. In that case, it makes sense to get rid of the `master` build process. If you don't mind confirming that this is the case, I'll get rid of that part. Thanks!

Reply:
Exactly! It's a bit misleading, I know

Reply:
I've removed the production build. I think it's otherwise good-to-go. Thanks for letting me contribute. This was fun.